index.qmd

```diff
@@ -1,120 +1,212 @@
----
-title: "My Final Project Template"
-author: Your name here
-subtitle: Subtitle here if desired
-date: today
-date-format: long
```

Comment:
I would suggest folding your code so there is an option to view it. That can be done by adding the following code to your YAML header. 
html: 
  code-fold: true
  code-summary: "Show the code"